chainguard-dev/edu
# Reorganizing Chainguard Images docs

## Type of change

This PR moves around a bunch of the Chainguard Images docs with the goal being to improve their disocverability. It creates a few new buckets and I did my vest to group docs together in ways that make sense. 

I'm very open to suggestions!

### What should this PR do?

Resolves https://github.com/chainguard-dev/internal/issues/3556

### Why are we making this change?

Our Images docs are getting out of control, this PR aims to clean them up and make things easier to find.

### What are the acceptance criteria? 


@metalstormbass was the first to mention (to me anyway) that Images docs were hard to find. Would appreciate their feedback here.

### How should this PR be tested?

Make sure everything looks okay. I did my best to add aliases where necessary and added an entry to the ngnix.conf to add a redirect for the `vulnerability-comoparisons` bucket.
